examples/runSingleFrameTask.py

```diff
@@ -22,27 +22,27 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
+from __future__ import print_function
 import os
 import sys
 import numpy
 
-import eups
-
-import lsst.daf.base               as dafBase
-import lsst.afw.table              as afwTable
-import lsst.afw.image              as afwImage
-import lsst.afw.display.ds9        as ds9
-import lsst.meas.algorithms        as measAlg
+from lsst.utils import getProductDir
+import lsst.afw.table as afwTable
+import lsst.afw.image as afwImage
+import lsst.afw.display.ds9 as ds9
+import lsst.meas.algorithms as measAlg
 from lsst.meas.algorithms.detection import SourceDetectionTask
 from lsst.meas.base import SingleFrameMeasurementTask
 
+
 def loadData():
     """Prepare the data we need to run the example"""
 
     # Load sample input from disk
-    mypath = eups.productDir("afwdata")
+    mypath = getProductDir("afwdata")
```

Comment:
Likewise. I thought somebody had eliminated all of them. That reminds me; since utils is directly called in the example I will add it to the required packages (it could be optional, but it must exist anyway so that seems a bit silly)